testcases/PowerNVDump.py

```diff
@@ -298,6 +298,7 @@ def verify_fadump_reg(self):
             self.c.run_command("dmesg > /tmp/dmesg_log")
             self.c.run_command("%s > /tmp/opal_log" % BMC_CONST.OPAL_MSG_LOG)
         self.c.run_command("echo 1 > /sys/kernel/fadump_registered")
+        self.c.run_command("cat /sys/kernel/fadump_registered")
```

Comment:
can you please split this in another commit

Reply:
It's done . It will be added in  next PR